environment/providers/kubernetes/kubernetes.go

```diff
@@ -5,14 +5,29 @@
 package kubernetes
 
 import (
-	"encoding/json"
 	"fmt"
 	"net/url"
 
+	"github.com/nutanix-cloud-native/prism-go-client/environment/credentials"
 	"github.com/nutanix-cloud-native/prism-go-client/environment/types"
 	coreinformers "k8s.io/client-go/informers/core/v1"
 )
 
+// Type aliases for backward compatibility
+type (
+	CredentialType             = credentials.CredentialType
+	NutanixCredentials         = credentials.NutanixCredentials
+	BasicAuthCredential        = credentials.BasicAuthCredential
+	BasicAuth                  = credentials.BasicAuth
+	PrismCentralBasicAuth      = credentials.PrismCentralBasicAuth
+	PrismElementBasicAuth      = credentials.PrismElementBasicAuth
+	NutanixCredentialKind      = credentials.NutanixCredentialKind
+	NutanixCredentialReference = credentials.NutanixCredentialReference
+	NutanixPrismEndpoint       = credentials.NutanixPrismEndpoint
+)
```

Comment:
I don't think we need these since we didn't check them in `0.2.0` and `0.3.0` hasn't been released yet

Reply:
CCM is using these as consumer of library.

Reply:
CCM can update to using `credentials.X` next time they update the lib version. Good thing go modules is that you can pin deps to git hashes so it won't automatically break.

Reply:
True I can simplify change here.

Reply:
Just to be clear we break backward compatibility by not doing type aliases.